__tests__/queryMock-tests.js

```diff
@@ -206,15 +206,22 @@ describe('queryMock', () => {
     beforeEach(() => {
       mockData = { some: 'data' };
       mockVariables = {
-        includeStuff: true,
-        withStuff: true,
-        filterOn: [123, 345],
-        after: 'cursor'
+        firstParam: [
```

Comment:
Would you mind leaving the old tests _as-is_ and just add new ones for the object property ordering?